README.md

```diff
@@ -75,6 +75,16 @@ To enable Google Anaytics, add the following lines to your Jekyll site:
 
 Google Analytics will only appear in production, i.e., `JEKYLL_ENV=production`
 
+--
+
+### Enabling Excerpts on the Home Page
+
+To display post-excerpts on the Home Page, simply add the following to your `_config.yml`:
+
+```yaml
+show_excerpts: enabled
```

Comment:
Should excerpts just be enabled in all cases? Enabled by default? Why do we need to make this an option?